docker/test/performance-comparison/compare.sh

```diff
@@ -7,6 +7,10 @@ trap 'kill $(jobs -pr) ||:' EXIT
 stage=${stage:-}
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+# https://github.com/jemalloc/jemalloc/wiki/Getting-Started
+export MALLOC_CONF="percpu_arena:disabled"
+echo "$MALLOC_CONF" > /etc/malloc.conf ||:
```

Comment:
```suggestion
ln -s "percpu_arena:disabled" > /etc/malloc.conf
```

This should be a symlink location, not the content of the file

https://github.com/jemalloc/jemalloc/blob/259c5e3e8f4731f2e32ceac71c66f4bc7d078145/src/jemalloc.c#L1052